fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -127,6 +131,14 @@ public void contextInitialized(ServletContextEvent event) {
             log.fine("Initializing FHIRRegistry...");
             FHIRRegistry.getInstance();
 
+            Boolean serverRegistryResourceProviderEnabled = fhirConfig.getBooleanProperty(PROPERTY_SERVER_REGISTRY_RESOURCE_PROVIDER_ENABLED, Boolean.FALSE);
+            if (serverRegistryResourceProviderEnabled) {
+                log.info("Registering ServerRegistryResourceProvider...");
+                ServerRegistryResourceProvider provider = new ServerRegistryResourceProvider(persistenceHelper);
+                FHIRRegistry.getInstance().register(provider);
```

Comment:
maybe a comment here to explain that getInstance() loads all the static providers

Reply:
Added Javadoc